### DISCUSSION TITLE
Feat: Support CLI completions in Docker containers

### DISCUSSION THREAD

**Discussion by brendanfalk** [aws/amazon-q-developer-cli#208] [General]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Feature Details

A lot of users run Docker containers locally and run an interactive shell to interact with them. We should support fast CLI completions on these machines.

If you have a specific machine you want support for, please comment!

**Comment by alanhe421** [comment]
I'm thinking that you still need to load the profile file in the container if you want to support it, otherwise you can't determine the PS1 and command contents.

**Reply by brendanfalk** [reply]
Agree!
